src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+    private Color[] colors = Color.values();
```

Comment:
The `random` and `colors` fields could be made final, as they are not expected to be changed after being initialized.

Reply:
Done it but the same question.